### PR TITLE
Quiz Engine

### PR DESCRIPTION
**Pull request by varun13169**

### CODE REVIEW

File: src/backend/db/quizCategory.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { v4 as uuid } from "uuid";
+
+const quizeCategories = {
```
Comment (iRohitGaur) [inline src/backend/db/quizCategory.js:3]: suggestion: I think `quizCategories` got misspelled

File: src/page/categoryPage/CategoryPage.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { useEffect } from "react";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import { quizeCategories } from "../../backend/db/quizCategory";
+import { Navbar } from "../../components";
+import "./category-page.css";
+// import categoryPageStyles from "./category-page.modules.css";
```
Comment (iRohitGaur) [inline src/page/categoryPage/CategoryPage.jsx:7]: remove commented code

File: src/page/categoryPage/CategoryPage.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  const [categories, setCategories] = useState([]);
+
+  useEffect(() => {
+    setCategories((categories) => quizeCategories.categories);
+  }, []);
+
+  return (
+    <section className="category-page-namespace page-wrap">
+      <section className="category-page-namespace page-nav">
+        <Navbar></Navbar>
```
Comment (iRohitGaur) [inline src/page/categoryPage/CategoryPage.jsx:19]: suggestion: you can use self closing tag here like this: `<Navbar />`

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge with changes 👍🏻

File: src/backend/db/quizCategory.js (hunk 1 above)
Comment (sanjitsarkar) [reply src/backend/db/quizCategory.js:3]: Yes Fix the typo

**Review by sanjitsarkar — APPROVED** [review]

File: src/backend/db/quizCategory.js (hunk 1 above)
Comment (varun13169) [reply src/backend/db/quizCategory.js:3]: Done, in the upcoming PR

File: src/page/categoryPage/CategoryPage.jsx (hunk 3 above)
Comment (varun13169) [reply src/page/categoryPage/CategoryPage.jsx:19]: Done, in the upcoming PR

File: src/page/categoryPage/CategoryPage.jsx (hunk 2 above)
Comment (varun13169) [reply src/page/categoryPage/CategoryPage.jsx:7]: Done, in the upcoming PR

**Pull request merged**